core/domain/feedback_services.py

```diff
@@ -365,7 +365,8 @@ def get_total_open_threads(feedback_analytics_list):
     FeedbackAnalytics domain objects.
 
     Args:
-        feedback_analytics_list: list(FeedbackAnalytics).
+        feedback_analytics_list: list(FeedbackAnalytics). list of
```

Comment:
Why is the check not catching this? The description should start with a capital letter. 

A list of....

Reply:
The checks are still running on the codebase when, that is complete I will change all these to start with a capital letter.

Reply:
Done.

Reply:
In that case -- You should not request for a review until your PR is ready.